requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Python 3.10 is not supported yet in Specter, this is probably why all your tests are failing. Use pyenv to manage python versions.